cephfs/admin/pin.go

```diff
@@ -0,0 +1,40 @@
+//go:build !nautilus && ceph_preview
+// +build !nautilus,ceph_preview
+
+package admin
+
+// PinSubVolume pins subvolume to ranks according to policies.
+//
+// Similar To:
+//
+//	ceph fs subvolume pin <vol_name> <sub_name> <pin_type> <pin_setting>
+func (fsa *FSAdmin) PinSubVolume(volume, subvolume, Type string, Setting string) (string, error) {
```

Comment:
```suggestion
func (fsa *FSAdmin) PinSubVolume(volume, subvolume, type, setting string) (string, error) {
```
Function arguments are not usually capitalized IIRC. It is interesting that the check tools didn't detect this.

Reply:
Now I came to know that `type` is a reserved keyword 🤣 Going with `pintype` and `pinsetting`.